### ISSUE TITLE
Underlying connector does not support size() 

### ISSUE THREAD

**Issue by neel** [matthijs/sqlpp11-connector-postgresql#80]
Cannot get number of records returned while executing a select query.

**Comment by neel** [comment]
Any explanations ? Why is it closed ?

**Comment by matthijs** [comment]
Sorry, I should've commented on this bug.

I have made the size function available in the commit [d48039d](https://github.com/matthijs/sqlpp11-connector-postgresql/commit/d48039db9874c3d6a9e01130fd48ed538ab2b167)

Now it is possible to get the size on the result set like:
```c++
const auto test_size = db(select(all_of(tab)).from(tab).unconditionally());
std::cout << "Result set size: " << test_size.size() << std::endl;
```

**Comment by neel** [comment]
Yes checked that.  So I deleted that comment.
